src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
 public class Application {
+    public static final int AMOUNT_OF_CALLS = 3;
```

Comment:
1. better name
2. constant is not used outside, so can be private
```suggestion
    private static final int BALLS_TO_GENERATE = 3;
```